### ISSUE TITLE
'Show more' link not visible in all slots in month view 

### ISSUE THREAD

**Issue by sagarj92** [jquense/react-big-calendar#1775]
#### Do you want to request a _feature_ or report a _bug_?
 a bug
#### What's the current behavior?
show more link is not visible for some time slots, even though they are available in feeding data. Please see attached sandbox where for **18, 19, 20th August there are 35-40** events available.
[https://codesandbox.io/s/lucid-dhawan-hvddg](https://codesandbox.io/s/lucid-dhawan-hvddg)

#### What's the expected behavior?
show more links should be shown for applicable time slots.

**Comment by bennyfink** [comment]
Looks like it works properly in the sandbox now.

**Comment by feelinggeeky** [comment]
> #### Do you want to request a _feature_ or report a _bug_?
> a bug
> 
> #### What's the current behavior?
> show more link is not visible for some time slots, even though they are available in feeding data. Please see attached sandbox where for **18, 19, 20th August there are 35-40** events available still no show more link visible.
> https://codesandbox.io/s/lucid-dhawan-hvddg
> 
> #### What's the expected behavior?
> show more links should be shown for applicable time slots.

I have the same problem, and I cannot find any solution to solve it, I'm stuck here...waiting for SOS

**Comment by bennyfink** [comment]
@feelinggeeky Can you go into more detail about your specific problem? Does the solution on #1213 help your situation?

**Comment by feelinggeeky** [comment]
@bennyfink For example, there are 6 schedule events, the first five are schedules from 13th to 15th, and the last schedule is from 14th to 16th, so the last schedule will be displayed at the bottom. When the height of all event content is obviously greater than the line height, show more should be displayed. However, in my case, show more on the 16th disappeared. I set the height of .rbc-event and it had no effect.

**Comment by gnapse** [comment]
I was having issues similar to this. Specifically, the "show more" button would be rendered, but not visible, because the calendar was rendering more events in a day cell than it could fit, so the "show more" button was cut off, but it was there.

Upon inspection, I found that the issue for me was that I was rendering a custom event wrapper that was making each event slightly taller than the default. It was a line-height. Once I reduced the line-height, it all worked as expected again.

So my question is: aren't we supposed to render a custom event that takes more height than the default?

**Comment by legnaib** [comment]
Yes, I have the same issue! When having custom events spanning more than one line, this is not taken into account and the events and "show more" button just overflows. Instead it should reduce the number of events and show the show more button earlier. A custom method when to show the show more button would be helpful
